internal/server/data/migrations_test.go

```diff
@@ -151,12 +150,10 @@ func TestMigrations(t *testing.T) {
 			},
 		},
 		{
-			// this test does an external call to example.okta.com, if it fails check your network connection
 			label: testCaseLine("202206281027"),
 			setup: func(t *testing.T, db *gorm.DB) {
 				stmt := `
 INSERT INTO providers (id, created_at, updated_at, deleted_at, name, url, client_id, client_secret, kind, created_by) VALUES (67301777540980736, '2022-07-05 17:13:14.172568+00', '2022-07-05 17:13:14.172568+00', NULL, 'infra', '', '', 'AAAAEIRG2/PYF2erJG6cYHTybucGYWVzZ2NtBDjJTEEbL3Jvb3QvLmluZnJhL3NxbGl0ZTMuZGIua2V5DGt4MdtlZuxOUhZQTw', 'infra', 1);
-INSERT INTO providers (id, created_at, updated_at, deleted_at, name, url, client_id, client_secret, kind, created_by) VALUES (67301777540980737, '2022-07-05 17:13:14.172568+00', '2022-07-05 17:13:14.172568+00', NULL, 'okta', 'example.okta.com', 'client-id', 'AAAAEIRG2/PYF2erJG6cYHTybucGYWVzZ2NtBDjJTEEbL3Jvb3QvLmluZnJhL3NxbGl0ZTMuZGIua2V5DGt4MdtlZuxOUhZQTw', 'okta', 1);
```

Comment:
If we remove this, we are no longer testing the migration. We can't show that a non-infra provider has a kind set.  Can we keep it?

Reply:
Kind being set is tested separately in `202206151027`. I'll add this back for now though, this migration will probably be removed soon anyway 👍

Reply:
Brought it back